### PR TITLE
Files restruct

### PR DESCRIPTION
**Pull request by MaxDrozdov**

### CODE REVIEW

File: src/app/routing/app.routing.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+(function () {
+
+     angular.module('base.app').config(function ($routeProvider) {
+
+          $routeProvider.when('/home', {
+               templateUrl: 'news/view/news.view/news.html',
+               controller: 'newsCtrl',
```
Comment (strellka) [inline src/app/routing/app.routing.js]: redundant comma, remove it please

File: src/app/routing/app.routing.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+          });
+
+          $routeProvider.when('/settings', {
+               templateUrl: 'settings/view/settings.view.html',
+               controller: 'settingsCtrl',
```
Comment (strellka) [inline src/app/routing/app.routing.js]: same

File: src/app/routing/app.routing.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+          });
+
+          $routeProvider.when('/tape', {
+               templateUrl: 'app/view/404.page/404.html',
+               controller: '',
+          });
```
Comment (strellka) [inline src/app/routing/app.routing.js]: same

File: src/app/routing/app.routing.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+          $routeProvider.when('/articles', {
+               templateUrl: 'app/view/404.page/404.html',
+               controller: '',
```
Comment (strellka) [inline src/app/routing/app.routing.js]: same

File: src/app/routing/app.routing.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+          });
+
+          $routeProvider.when('/rating', {
+               templateUrl: 'app/view/404.page/404.html',
+               controller: '',
```
Comment (strellka) [inline src/app/routing/app.routing.js]: same

File: src/app/routing/app.routing.js (hunk 1 above)
Comment (MaxDrozdov) [inline src/app/routing/app.routing.js]: Done.

File: src/app/routing/app.routing.js (hunk 2 above)
Comment (MaxDrozdov) [inline src/app/routing/app.routing.js]: Done.

File: src/app/routing/app.routing.js (hunk 3 above)
Comment (MaxDrozdov) [inline src/app/routing/app.routing.js]: Done.

File: src/settings/style/style.settings.css [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+.style-menu>li.active>a, .style-menu>li.active>a:hover, .style-menu>li.active>a:focus{
+     background-color: #000000;
+}
+
+.style-menu>li>a:hover, .style-menu>li>a:focus{
+     color: #ffffff;
+     background-color: #000000;
+}
```
Comment (strellka) [inline src/settings/style/style.settings.css]: fix codestyle in this file

File: src/app/routing/app.routing.js (hunk 4 above)
Comment (MaxDrozdov) [inline src/app/routing/app.routing.js]: Done.

File: src/app/routing/app.routing.js (hunk 5 above)
Comment (MaxDrozdov) [inline src/app/routing/app.routing.js]: Done.

File: src/settings/view/settings.view.html [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+                                                                            ng-click="setActivePage('about')">About</a>
+                    </li>
+               </ul>
+          </div>
+
+          <div class="col-lg-9 col-md-9 col-sm-9 col-xs-12">
+
+          </div>
+     </div>
+</div>
```
Comment (strellka) [inline src/settings/view/settings.view.html]: fix codestyle

Comment (MaxDrozdov) [inline src/settings/view/settings.view.html]: Done.

File: src/settings/style/style.settings.css (hunk 6 above)
Comment (MaxDrozdov) [inline src/settings/style/style.settings.css]: Done.

**Pull request merged**